sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/CoverageResultImporter.java

```diff
@@ -144,18 +140,10 @@ private static void saveCoverageLineHitsData(FileNode fileNode, InputFile inputF
    * @return the coverage
    */
   private CoverageNode getCoverage(File coverageReportFile) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "getCoverage" a "static" method. ([squid:S2325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.php%3Aphp&branch=SONARPHP-758&issues=AWFIRjS0zrLehFnPs_1e&open=AWFIRjS0zrLehFnPs_1e)

[ISSUE_KEY]: AWFIRjS0zrLehFnPs_1e